tests/models/test_dag.py

```diff
@@ -1403,6 +1404,18 @@ def test_create_dagrun_job_id_is_set(self):
         )
         assert dr.creating_job_id == job_id
 
+    def test_dag_add_task_sets_default_task_group(self):
+        dag = DAG(dag_id="test_dag_add_task_sets_default_task_group", start_date=DEFAULT_DATE)
+        task_without_task_group = EmptyOperator(task_id="task_without_group_id")
+        default_task_group = TaskGroupContext.get_current_task_group(dag)
+        dag.add_task(task_without_task_group)
+        assert default_task_group.get_child_by_label("task_without_group_id") is not None
```

Comment:
Instead of `is not None` is it better to explicitly compare it to `dag.get_task("task_without_group_id")`?